string room_name
---
string medication